src/test/java/step3/domain/WinnerLottoTest.java

```diff
@@ -1,21 +1,49 @@
 package step3.domain;
 
 import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
 
 import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
 class WinnerLottoTest {
+
+    final WinnerLotto winnerLotto = new WinnerLotto(Arrays.asList(1, 2, 3, 4, 5, 6), 10);
+
     @Test
-    void valid_1등_로또_생성() {
-        WinnerLotto winnerLotto = new WinnerLotto(Arrays.asList(1, 2, 3, 4, 5, 6));
-        assertThat(winnerLotto).isEqualTo(new WinnerLotto(Arrays.asList(1, 2, 3, 4, 5, 6)));
+    void valid_당첨로또생성() {
+        assertThat(winnerLotto).isEqualTo(new WinnerLotto(Arrays.asList(1, 2, 3, 4, 5, 6), 10));
     }
 
+    @DisplayName("당첨로또 생성 시, 입력된 숫자가 6개를 초과하면 IllegalArgumentException을 발생시킨다.")
```

Comment:
만약 CustomException 을 발생시키게 되면 테스트 설명 또한 변경해줘야 할 것 같아요.
테스트의 설명에 구현을 포함하기 보다는 요구사항을 중심으로 작성해 보면 어떨까요? 😄 
```suggestion
    @DisplayName("당첨 로또 번호는 6개를 초과할 수 없다.")
```

Reply:
메시지 작성하는게 정말 어려운 것 같습니다. 😄 
다음 단계에 반영하겠습니다!